demo/client does not compile

> Windows 10 64 bits
>
> node v8.4.0
> npm 5.3.0
> webpack 3.5.6

built with the following commands :

```
npm install
npm run build
```

```
Version: webpack 3.5.6
Time: 81ms
             Asset     Size  Chunks             Chunk Names
      text-demo.js  6.18 kB       0  [emitted]  text-demo
       svg-demo.js  6.21 kB       1  [emitted]  svg-demo
  mesh-debugger.js  6.15 kB       2  [emitted]  mesh-debugger
        3d-demo.js  6.14 kB       3  [emitted]  3d-demo
      3d-demo.html  6.64 kB          [emitted]
        index.html  4.14 kB          [emitted]
mesh-debugger.html  3.81 kB          [emitted]
     svg-demo.html   8.1 kB          [emitted]
    text-demo.html  10.5 kB          [emitted]
   [0] ./src/3d-demo.ts 329 bytes {3} [built] [failed] [1 error]
   [1] ./src/svg-demo.ts 385 bytes {1} [built] [failed] [1 error]
   [2] ./src/text-demo.ts 355 bytes {0} [built] [failed] [1 error]
   [3] ./src/mesh-debugger.ts 327 bytes {2} [built] [failed] [1 error]

ERROR in ./src/3d-demo.ts
Module parse failed: D:\Workspace\pathfinder\demo\client\src\3d-demo.ts Unexpected token (28:11)
You may need an appropriate loader to handle this file type.
| import PathfinderBufferTexture from "./buffer-texture";
|
| const WIDTH: number = 150000;
| const PADDING: number = 2000;
|

ERROR in ./src/mesh-debugger.ts
Module parse failed: D:\Workspace\pathfinder\demo\client\src\mesh-debugger.ts Unexpected token (25:15)
You may need an appropriate loader to handle this file type.
| import * as opentype from "opentype.js";
|
| const CHARACTER: string = 'r';
|
| const FONT: string = 'eb-garamond';

ERROR in ./src/svg-demo.ts
Module parse failed: D:\Workspace\pathfinder\demo\client\src\svg-demo.ts Unexpected token (28:12)
You may need an appropriate loader to handle this file type.
| const parseColor = require('parse-color');
|
| const SVG_NS: string = "http://www.w3.org/2000/svg";
|
| const PARTITION_SVG_PATHS_ENDPOINT_URL: string = "/partition-svg-paths";

ERROR in ./src/text-demo.ts
Module parse failed: D:\Workspace\pathfinder\demo\client\src\text-demo.ts Unexpected token (32:18)
You may need an appropriate loader to handle this file type.
| import SSAAStrategy from './ssaa-strategy';
|
| const DEFAULT_TEXT: string =
| `’Twas brillig, and the slithy toves
| Did gyre and gimble in the wabe;
```

Yes, I have to build the client in a Linux VM. 

With the latest version of pathfinder on my WIn10 64bits :

```
PS D:\Workspace\pathfinder\demo\client> npm -v
5.4.2
```

```
PS D:\Workspace\pathfinder\demo\client> node -v
v8.6.0

```


```
PS D:\Workspace\pathfinder\demo\client> npm install

> uglifyjs-webpack-plugin@0.4.6 postinstall D:\Workspace\pathfinder\demo\client\node_modules\uglifyjs-webpack-plugin
> node lib/post_install.js

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN pathfinder-demo@0.1.0 No repository field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.2 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

added 453 packages in 27.454s
```


```
PS D:\Workspace\pathfinder\demo\client> npm run build

> pathfinder-demo@0.1.0 build D:\Workspace\pathfinder\demo\client
> webpack

HandlebarsPlugin: + helper 'octicon'
HandlebarsPlugin: start compilation
HandlebarsPlugin: + partial 'partials/header.html'
HandlebarsPlugin: + partial 'partials/navbar.html'
HandlebarsPlugin: + partial 'partials/spinner.html'
HandlebarsPlugin: created output './3d-demo.html'
HandlebarsPlugin: created output './benchmark.html'
HandlebarsPlugin: created output './index.html'
HandlebarsPlugin: created output './mesh-debugger.html'
HandlebarsPlugin: created output './svg-demo.html'
HandlebarsPlugin: created output './text-demo.html'

Hash: 8623da1e1e2920a7ded4
Version: webpack 3.6.0
Time: 112ms
             Asset     Size  Chunks             Chunk Names
      3d-demo.html   7.5 kB          [emitted]
      text-demo.js   6.2 kB       0  [emitted]  text-demo
  mesh-debugger.js  6.15 kB       2  [emitted]  mesh-debugger
      benchmark.js  6.23 kB       3  [emitted]  benchmark
        3d-demo.js  6.15 kB       4  [emitted]  3d-demo
       svg-demo.js  6.17 kB       1  [emitted]  svg-demo
    benchmark.html  7.12 kB          [emitted]
        index.html  4.25 kB          [emitted]
mesh-debugger.html  7.99 kB          [emitted]
     svg-demo.html  9.03 kB          [emitted]
    text-demo.html  11.7 kB          [emitted]
   [0] ./src/3d-demo.ts 335 bytes {4} [built] [failed] [1 error]
   [1] ./src/svg-demo.ts 348 bytes {1} [built] [failed] [1 error]
   [2] ./src/text-demo.ts 382 bytes {0} [built] [failed] [1 error]
   [3] ./src/benchmark.ts 405 bytes {3} [built] [failed] [1 error]
   [4] ./src/mesh-debugger.ts 325 bytes {2} [built] [failed] [1 error]

ERROR in ./src/3d-demo.ts
Module parse failed: D:\Workspace\pathfinder\demo\client\src\3d-demo.ts Unexpected token (30:26)
You may need an appropriate loader to handle this file type.
| import {DemoView, Timings} from "./view";
|
| const TEXT_AVAILABLE_WIDTH: number = 150000;
| const TEXT_PADDING: number = 2000;
|

ERROR in ./src/benchmark.ts
Module parse failed: D:\Workspace\pathfinder\demo\client\src\benchmark.ts Unexpected token (30:12)
You may need an appropriate loader to handle this file type.
| import {DemoView, MonochromeDemoView, Timings } from "./view";
|
| const STRING: string = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
|
| const FONT: string = 'nimbus-sans';

ERROR in ./src/mesh-debugger.ts
Module parse failed: D:\Workspace\pathfinder\demo\client\src\mesh-debugger.ts Unexpected token (29:15)
You may need an appropriate loader to handle this file type.
| import {PathfinderView} from "./view";
|
| const CHARACTER: string = 'A';
|
| const FONT: string = 'eb-garamond';

ERROR in ./src/svg-demo.ts
Module parse failed: D:\Workspace\pathfinder\demo\client\src\svg-demo.ts Unexpected token (30:12)
You may need an appropriate loader to handle this file type.
| const parseColor = require('parse-color');
|
| const SVG_NS: string = "http://www.w3.org/2000/svg";
|
| const DEFAULT_FILE: string = 'tiger';

ERROR in ./src/text-demo.ts
Module parse failed: D:\Workspace\pathfinder\demo\client\src\text-demo.ts Unexpected token (35:18)
You may need an appropriate loader to handle this file type.
| import {DemoView, MonochromeDemoView, Timings, TIMINGS} from './view';
|
| const DEFAULT_TEXT: string =
| `’Twas brillig, and the slithy toves
| Did gyre and gimble in the wabe;
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! pathfinder-demo@0.1.0 build: `webpack`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the pathfinder-demo@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\quent\AppData\Roaming\npm-cache\_logs\2017-10-07T15_11_48_637Z-debug.log
```

Same is here.Trying to build the client and getting this:

> $ npm run build
> 
> > pathfinder-demo@0.1.0 build D:\pathfinder\demo\client
> > webpack
> 
> D:\pathfinder\demo\client\node_modules\handlebars-webpack-plugin\index.js:30
> class HandlebarsPlugin {
> ^^^^^
> 
> SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
>     at exports.runInThisContext (vm.js:53:16)
>     at Module._compile (module.js:413:25)
>     at Object.Module._extensions..js (module.js:452:10)
>     at Module.load (module.js:355:32)
>     at Function.Module._load (module.js:310:12)
>     at Module.require (module.js:365:17)
>     at require (module.js:384:17)
>     at Object.<anonymous> (D:\pathfinder\demo\client\webpack.config.js:2:26)
>     at Module._compile (module.js:434:26)
>     at Object.Module._extensions..js (module.js:452:10)
> 
> npm ERR! Windows_NT 10.0.15063
> npm ERR! argv "D:\\emsdk\\node\\4.1.1_64bit\\bin\\node.exe" "D:\\emsdk\\node\\4.1.1_64bit\\bin\\node_modules\\npm\\bin\\npm-cli.js" "run" "build"
> npm ERR! node v4.1.1
> npm ERR! npm  v2.14.4
> npm ERR! code ELIFECYCLE
> npm ERR! pathfinder-demo@0.1.0 build: `webpack`
> npm ERR! Exit status 1
> npm ERR!
> npm ERR! Failed at the pathfinder-demo@0.1.0 build script 'webpack'.
> npm ERR! This is most likely a problem with the pathfinder-demo package,
> npm ERR! not with npm itself.
> npm ERR! Tell the author that this fails on your system:
> npm ERR!     webpack
> npm ERR! You can get their info via:
> npm ERR!     npm owner ls pathfinder-demo
> npm ERR! There is likely additional logging output above.
> 
> npm ERR! Please include the following file with any support request:
> npm ERR!     D:\pathfinder\demo\client\npm-debug.log
>

--- a/demo/client/webpack.config.js
+++ b/demo/client/webpack.config.js
@@ -14,7 +14,7 @@ module.exports = {
     module: {
         rules: [
             {
-                test: /src\/[a-zA-Z0-9_-]+\.tsx?$/,
+                test: /src(\/|\\)[a-zA-Z0-9_-]+\.tsx?$/,
                 enforce: 'pre',
                 loader: 'tslint-loader',
                 exclude: /node_modules/,
@@ -23,7 +23,7 @@ module.exports = {
                 },
             },
             {
-                test: /src\/[a-zA-Z0-9_-]+\.tsx?$/,
+                test: /src(\/|\\)[a-zA-Z0-9_-]+\.tsx?$/,
                 use: 'ts-loader',
                 exclude: /node_modules/,
             },

